policies/rules.md

```diff
@@ -114,19 +121,19 @@ By submitting a Submission, you grant the organizers, sponsor and the CODEVID-19
 
 Your Submission may be posted in public on the internet. We are not responsible for any unauthorized use of your submission or project by users of the CODEVID-19 website.
 
-### Communication Consent and Responsibilities
+## Communication Consent and Responsibilities
 
-By participating, participants agree to give organizers permission to contact them with information and communication relevant to CODEVID-19 via Slack, email, or other methods deemed appropriate by event organizers. This consent may be withdrawn at anytime be contacting contact@codevid19.com, or using the unsubscribe feature in the mailing list.
+By participating, participants agree to give organizers permission to contact them with information and communication relevant to CODEVID-19 via Slack, email, or other methods deemed appropriate by event organizers. This consent may be withdrawn at any time be contacting contact@codevid19.com or using the unsubscribe feature in the mailing list.
```

Comment:
I just noticed that right after `any time` the `be` should be `by`.

Reply:
Small absolutely clear-as-day typos like this will be part of this PR.  Anything that's subjective I'll keep out.